backend/src/libs/packages/config/config.package.ts

```diff
@@ -69,6 +69,15 @@ class Config implements IConfig {
           default: null,
         },
       },
+      JWT: {
+        SECRET: {
+          doc: 'Secret key for token generation',
+          format: String,
+          env: 'ACCESS_TOKEN_SECRET',
+          default: null,
+        },
+        EXPIRES_IN: '24h',
```

Comment:
Instead of injecting it to env and faking the fact of getting it from env - make method to get auth config
 ```
 private get authConfig(): AuthConfig {
    return {
      ALGORITHM: 'HS256',
      EXPIRES_IN: '24h'
    };
  }
  ```
  and define in `constructor` values for ENCRYPTION
  ```
    this.AUTH = this.authConfig;
  ```

```suggestion

```